mgmt/src/rpc/overlay/vpc.rs

```diff
@@ -12,6 +12,8 @@ use std::collections::BTreeSet;
 
 use crate::rpc::overlay::VpcManifest;
 use crate::rpc::overlay::VpcPeeringTable;
+#[cfg(doc)]
```

Comment:
why is this needed?

Reply:
@daniel-noland , sorry I forgot to review this. I'm not quite sure about the motivation, but you can go ahead and merge if it's blocking you.

Reply:
because this import is only used in the doc comments